src/components/Select/SelectT.tsx

```diff
@@ -0,0 +1,258 @@
+import classNames from 'classnames';
```

Comment:
@constanzauanini this is basically the original `Select` component but string value type has been made a generic `T`.